specs.js

```diff
@@ -0,0 +1,153 @@
+/* global Given, Then, describe */
+var expect = require('chai').expect
+var nock = require('nock')
+
+describe('when using trimperson api', () => {
+  var trim
+  var mockURL
+  var mockToken
+
+  Given(() => mockURL = 'http://localhost')
+  Given(() => mockToken = 'DEADBEEF')
+
+  Given(() => {
+    trim = require('./index')({
+      url: mockURL,
+      token: mockToken,
+      mock: false
+    })
+  })
+
+  Then('it should have created a trimperson object', () => expect(trim).to.be.a('object'))
+
+  describe('using the getContainer function', () => {
+    Then('the function should exist on the trim instance', () => expect(trim.getContainer).to.be.a('function'))
+
+    describe('when invoking the getContainer funciton', () => {
+      var getContainerMock
+      var fakeID = 'fakecontainerid'
+
+      Given(() => {
+        getContainerMock = nock(mockURL)
+          .get('/GetContainer')
+          .query({trimid: fakeID, securityToken: mockToken})
+          .reply(200, {
+            containerNo: 'someid',
+            subContainers: []
+          })
+      })
+
+      Then('it should resolve without error', (done) => {
+        trim.getContainer(fakeID, (err, data) => {
+          expect(err).not.to.exist
+          expect(data).to.exist
+
+          expect(data.containerNo).to.equal('someid')
+          expect(data.subContainers).to.be.a('array')
+          getContainerMock.done() // throws an error if no request was recorded
+          done()
+        })
+      })
+    })
+  })
+
+  describe('when using the getDocument function', () => {
+    var documentId
+    var getDocumentMock
+    Given(() => documentId = 'someIDthatPointsToADocument')
+    Given(() => {
+      getDocumentMock = nock(mockURL)
+        .get('/get')
+        .query({id: documentId, securityToken: mockToken})
+        .reply(200, 'data:video/mp4;base64,R0lGOD lhCwAOAMQfAP////7+/vj4+Hh4eHd3d/v7+/Dw8')
+    })
+
+    Then('it should getDocument succesfully', (done) => {
+      trim.getDocument(documentId, function (err, data) {
+        expect(err).not.to.exist
+        expect(data).to.exist
+
+        getDocumentMock.done()
+        done()
+      })
+    })
+  })
+
+  describe('when using the createContainer function', () => {
+    var createContainerMock, folder, parentFolder, privacySetting
+    Given(() => folder = 'someFolder')
+    Given(() => parentFolder = 'someParentFolder')
```

Comment:
you should be able to use And here I think if you wish though I'm perfectly happy with multiple Given statements.